business_requirement_deliverable_cost/models/business.py

```diff
@@ -139,7 +161,7 @@ def action_button_update_estimation(self):
                 for resource in deliverable.resource_ids:
                     pricelist_id = resource._get_pricelist()
                     partner_id = resource._get_partner()
-                    resource.sale_price_unit = resource.product_id.lst_price
+                    resource.sale_price_unit = resource.product_id.list_price
```

Comment:
@elicoidal  I am sorry but not sure what you mean with `list_price` and `price`. Maybe you are asking why I don't use `price` on this situation?